frontend/components/SamplesPage/StepSlider.tsx

```diff
@@ -23,7 +23,30 @@ type StepSliderProps = {
 const StepSlider: FunctionComponent<StepSliderProps> = ({onChange, value, steps}) => {
     const {t} = useTranslation('samples');
     const [step, setStep] = useState(value);
-    useEffect(() => setStep(value), [setStep, value]);
+    const timer = useRef(null as NodeJS.Timeout | null);
```

Comment:
`useRef<NodeJS.Timeout | null>` could be better

Reply:
good point